src/ManagedDripsHub.sol

```diff
@@ -128,9 +133,30 @@ abstract contract ManagedDripsHub is DripsHub, UUPSUpgradeable {
         _;
     }
 
-    /// @notice Gets the storage slot holding the paused flag.
-    function _pausedSlot() private pure returns (StorageSlot.BooleanSlot storage) {
-        return StorageSlot.getBooleanSlot(SLOT_PAUSED);
+    /// @notice Returns the DripsHub storage.
+    /// @return storageRef The storage.
+    function _dripsHubStorage()
+        internal
+        view
+        override
+        returns (DripsHubStorage storage storageRef)
+    {
+        return _managedDripsHubStorage().dripsHubStorage;
+    }
+
+    /// @notice Returns the ManagedDripsHub contract storage.
+    /// @return storageRef The storage.
+    function _managedDripsHubStorage()
+        internal
+        pure
+        returns (ManagedDripsHubStorage storage storageRef)
+    {
+        bytes32 slot = STORAGE_SLOT;
+        // solhint-disable-next-line no-inline-assembly
+        assembly {
+            // Based on OpenZeppelin's StorageSlot
+            storageRef.slot := slot
+        }
     }
```

Comment:
@CodeSandwich What do you think about making `DripsHub` itself `UUPSUpgradeable` so we can remove `ManagedDripsHub` completely?

As I see it, the only reason that would speak for not doing that would be that we need to be able to deploy a `DripsHub` instance without upgradeability.

Reply:
Agree.

> maybe even abstract DripsHub further and extract from it a few simpler libraries.

So, we could do a `SplitsLibrary` and `DripsLibrary`.